code/modules/clothing/suits/armor.dm

```diff
@@ -229,7 +229,7 @@
 			if(M == owner)
 				continue
 			owner.Beam(M,icon_state="lightning[rand(1, 12)]",icon='icons/effects/effects.dmi',time=5)
-			M.adjustFireLoss(25)
+			M.adjustFireLoss(10)
```

Comment:
Oh, well, make sure it at least factors in the target's defenses. (shields, armor, etc)